adoc/vi-Physical/SA.adoc

```diff
@@ -254,35 +260,33 @@ So disable the other hana variables with adding a '#' in front of them
 
 . Open the tfvars file again to make final changes.
 +
-To create a HANA Systemreplication HA automation uncomment
+To create a HANA Systemreplication HA automation, uncomment:
 
     #hana_ha_enabled = true
 +
-by removing the _#_
+by removing the _#_.
 +
-As now the system creates a cluster, we need to enable a few other services. Uncomment
+To create a cluster, we need to enable a few other services. Uncomment:
```

Comment:
This is imho wrong, as the definition to create a cluster was the step before. Because the switch is now ON we require  additional services, as this is not done automatically